tox.ini

```diff
@@ -62,16 +107,6 @@ commands =
     sphinx-build --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
     python -c 'import pathlib; print("Documentation available under file://\{0\}".format(pathlib.Path(r"{toxinidir}") / "docs" / "_build" / "index.html"))'
 
-[testenv:codestyle]
-pypi_filter =
-skip_install = true
-description = Run all style and file checks with pre-commit
-deps =
-    pre-commit
-commands =
-    pre-commit install-hooks
-    pre-commit run --color auto --all-files --show-diff-on-failure
-
 # Requires tox-conda
```

Comment:
@nabobalis are we using this or should it get deleted?

Reply:
it's possibly on the list, but it's quite hard to do generically. SunPy core has it's own env file, so not exactly portable.

Reply:
@CyclingNinja can you nuke the whole conda section?

Reply:
it's more that I don't like the duplication of the requirements, but maybe we can't find a better way.

Reply:
The requirements in the conda env file?